shell/src/main/java/org/apache/accumulo/shell/commands/EGrepCommand.java

```diff
@@ -31,7 +30,7 @@
 
   @Override
   protected void setUpIterator(final int prio, final String name, final String term,
-      final BatchScanner scanner, CommandLine cl, boolean negate) throws IOException {
+      final BatchScanner scanner, CommandLine cl, boolean negate, final Shell shellState) throws Exception {
```

Comment:
Seems like `shellState` is not used.

Reply:
correct again, removed shellState from method signature